damus/Views/Muting/MutelistView.swift

```diff
@@ -104,6 +90,20 @@ struct MutelistView: View {
                     }
                 }
             }
+        Section(NSLocalizedString("Users", comment: "Section header title for a list of muted users.")) {
```

Comment:
Try this. Also your code indentation is off.

```suggestion
            Section(
                header: Text("Users", comment: "Section header title for a list of muted users."),
                footer: Text("").padding(.bottom, 10 + tabHeight + getSafeAreaBottom())
            ) {
```

And replace the Threads section above with:
```swift
            Section(NSLocalizedString("Threads", comment: "Section header title for a list of threads that are muted.")) {
```